api/src/queries/project/project-update-queries.ts

```diff
@@ -206,3 +206,36 @@ export const putProjectSQL = (
 
   return sqlStatement;
 };
+
+/**
+ * SQL query to get coordinator information, for update purposes.
```

Comment:
wrong comment here, should probably say:

```suggestion
 * SQL query to get objectives information, for update purposes.
```